tests/Rector/Architecture/DoctrineRepositoryAsService/Correct/correct2.php.inc

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Post;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+final class PostController extends Controller
+{
+    /**
+     * @var \AppBundle\Repository\PostRepository
+     */
+    private $postRepository;
+    public function __construct(\AppBundle\Repository\PostRepository $postRepository)
+    {
+        $this->postRepository = $postRepository;
+    }
+    public function somethingAction(int $id): Response
+    {
+        $em = $this->getDoctrine()->getManager();
```

Comment:
I thought this Rector should remove this method too, or not?
Or do you plan to add it in new class after this gets reviewed (that would be great :)) ?

Reply:
I don't know how to check if it's unused, that is why I kept it:)